examples/custom-email-templates/README.md

```diff
@@ -0,0 +1,16 @@
+# Custom email templates
+
+Nhost plans to fully host custom email templates for email verification, password reset, email change confirmation, passwordless authentication...
+
+In the meantime, it is possible to use tell the Hasura-auth service to look for templates in an external public location through HTTP requests.
+Here is a short example on how to proceed.
```

Comment:
example **of** how to proceed? 🤔

Reply:
I would put an example on how to use its own template from a public repository as part of the Nhost documentation.
This readme is only an example on how to use this very repository